modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -367,16 +367,27 @@ definition {
 	@ignore = "Test Stub"
 	@priority = 3
 	test ShowUnderlineEffectsIsDisabledByDefault {
+		task ("Given home page") {
+			Navigator.openURL();
+		}
 
-		//task ("Given When Then") {
+		task ("When access accessibility menu") {
+			AccessibilityMenu.openAccessibilityMenu();
+		}
 
-		// // Given home page
-		// // When access accessibility menu
-		// // Then show underline effects in links is toggled OFF state
-		// // assert body element does not have "c-prefers-link-underline" class
+		task ("Then show underline effects in links is toggled OFF state") {
+			AssertNotChecked.assertNotCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
 
-		//}
+		task ("assert body element does not have 'c-prefers-link-underline' class") {
+			var key_class = ${key_class};
```

Comment:
I feel like this is unnecessary variable declaration. There's no `key_class` initialized prior to this line.